src/brands/peapods/lib/PeaShareContent.js

```diff
@@ -1,7 +1,10 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import Grid from '@material-ui/core/Grid';
 import { makeStyles } from '@material-ui/core/styles';
+import CopyIcon from '@material-ui/icons/FileCopyOutlined';
+import CheckIcon from '@material-ui/icons/Check';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
```

Comment:
please add this npm module as a dependency

Reply:
https://github.com/siriwatknp/mui-treasury-deprecate/blob/master/package.json#L59
it is already added.